cmd/ndm_daemonset/app/command/start.go

```diff
@@ -28,6 +38,49 @@ import (
 
 //NewCmdStart starts the ndm controller
 func NewCmdStart() *cobra.Command {
+	var wg sync.WaitGroup
+
+	go func() {
```

Comment:
Ignore the previous comment, This routine should be started just after we start the controller.